e2e/utilities/longhorn-install.sh

```
@@ -23,19 +25,46 @@
+  install_longhorn_by_manifest "${LONGHORN_STABLE_MANIFEST_URL}"
+}
+
 install_longhorn(){
   create_longhorn_namespace
   install_backupstores
   if [[ "${LONGHORN_INSTALL_METHOD}" == "helm" ]]; then
-    LONGHORN_REPO_URI=${LONGHORN_REPO_URI:-"https://github.com/longhorn/longhorn.git"}
-    LONGHORN_REPO_DIR="${TMPDIR}/longhorn"
     install_longhorn_by_chart
   elif [[ "${LONGHORN_INSTALL_METHOD}" == "manifest" ]]; then
     generate_longhorn_yaml_manifest "${TF_VAR_tf_workspace}"
```

Shellcheck

[warning] TF_VAR_tf_workspace is referenced but not assigned.

(SC2154)